assets/font/ListCompare.svg

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Great icon! Did you do it yourself? I'm asking because if it comes from somewhere else than your own creativity, we will need to check the license associated with it.

Reply:
It's a modified version of "TextDiff" icon already in the assets.

Reply:
I see! Thank you 😁